docs/ml-tuning.md

```diff
@@ -62,7 +62,7 @@ To help construct the parameter grid, users can use the [`ParamGridBuilder`](api
 
 After identifying the best `ParamMap`, `CrossValidator` finally re-fits the `Estimator` using the best `ParamMap` and the entire dataset.
 
-## Example: model selection via cross-validation
+## Examples: model selection via cross-validation
```

Comment:
I'm OK with this standardization overall. Here you still have a `##` section though, was that intended?